MovieApp/MyCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+
+import UIKit
+import PureLayout
+
+class MyCollectionViewCell: UICollectionViewCell {
+    
+    static let cellIdentifier = "crewCellIdentifier"
+    
+    private let crewMemberNameLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+    
+    private let crewMemberPositionLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        contentView.addSubview(crewMemberNameLabel)
+        contentView.addSubview(crewMemberPositionLabel)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .top, withInset: 3)
+        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .leading, withInset: 0)
+        crewMemberPositionLabel.autoPinEdge(.top, to: .bottom, of: crewMemberNameLabel, withOffset: 3)
+        crewMemberPositionLabel.autoPinEdge(toSuperviewEdge: .leading, withInset: 0)
```

Comment:
The same comment I left for the `crewMemberNameLabel` applies to the `crewMemberPositionLabel`. You are missing trailing and bottom constraint.